README.md

```diff
@@ -1 +1,15 @@
-# Hello-Microverse
\ No newline at end of file
+# Hello-Microverse Project 
+
+The objectives of this project is to:
+
+Create branches with different base branches.
+Understand the concept of version control.
+Follow GitHub flow.
+Open pull requests in GitHub.
+Write descriptive, easy to understand commit messages.
+Write short, detailed, easy to understand descriptions of pull requests.
+Write a descriptive and easy to understand README in business English for every project submitted.
+Demonstrate an ability to submit a project for code review.
+
+
+Author: Jaymelfah
```

Comment:
Great job @Jaymelfah 🤝

- [ ] As specified in the project requirements, kindly use the correct readme template and customize it according. This is the [readme template](https://github.com/microverseinc/readme-template).